packages/cms-lib/index.js

```diff
@@ -3,11 +3,11 @@ const {
   checkAndWarnGitInclusion,
   getAndLoadConfigIfNeeded,
   getConfig,
-  getPortalId,
-  getPortalConfig,
```

Comment:
We could, I checked for usages and didn't find any, unless I'm missing an area.

Reply:
https://github.com/HubSpot/cms-event-registration/blob/f9fcbdde9b9e62c01ee21495a4f9278c12a61752/scripts/createHubDb.js#L2 is an example (in addition to the webpack plugin, which we have to be careful about because the version of `@hubspot/cms-lib` is not pinned). I think there is some usage internally as well. I think the alternative is bumping the major version.

Reply:
I see, thank you for that.  I'll add them back in